core/lib/src/rocket.rs

```diff
@@ -151,7 +151,7 @@ impl Rocket {
                     let mut stream = body.into_chunk_stream(4096);
                     while let Some(next) = stream.next().await {
                         futures::future::poll_fn(|cx| sender.poll_ready(cx)).await.expect("TODO.async client gone?");
-                        sender.send_data(next?).expect("send chunk");
+                        sender.send_data(next?).await.expect("send chunk");
```

Comment:
This was renamed/refactored in hyper. We should remove the `poll_ready` line above and move its TODO to this line.

Reply:
Done.